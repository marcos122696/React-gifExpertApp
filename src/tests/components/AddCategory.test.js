import React from 'react'
import { shallow } from 'enzyme';
import { AddCategory } from '../../components/AddCategory';

describe('Pruebas en el componenete <AddCategory />', () => {

    const setCategories = () => {};
    const wrapper = shallow( <AddCategory setCategories={ setCategories } />);

    test('Debe de mostar el componente correctamente', () => { 
        expect( wrapper ).toMatchSnapshot();
    });

    test('Debe de cambiar la caja de texto', () => {

        const input = wrapper.find('input');
        const value = 'Hola Mundo';
        input.simulate('change', { target: { value }});
    });



});