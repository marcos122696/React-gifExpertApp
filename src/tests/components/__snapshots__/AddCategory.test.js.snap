// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`Pruebas en el componenete <AddCategory /> Debe de mostar el componente correctamente 1`] = `
<form
  onSubmit={[Function]}
>
  <input
    onChange={[Function]}
    placeholder="Buscar..."
    type="text"
    value=""
  />
</form>
`;
